assembl/locale/assembl-v2/es.po

```diff
@@ -539,104 +527,98 @@ msgstr "Configuración del Encabezado de la página principal"
 #. You can decide wether the participant can distribute a single type of token (exclusive) or several types of token per proposal.
 #: messages.json
 msgid "administration.helpers.exclusive"
-msgstr ""
-"Puede decidir si el participante puede distribuir un solo tipo de Token "
-"(exclusivo) o varios tipos de Tokens por propuesta."
+msgstr "Puede decidir si el participante puede distribuir un solo tipo de Token (exclusivo) o varios tipos de Tokens por propuesta."
 
 #. [administration.helpers.gaugeVoteInstructions]
 #. defaultMessage is:
 #. Depending on the objective of the gauge module, incite the participants to take action.
 #: messages.json
 msgid "administration.helpers.gaugeVoteInstructions"
-msgstr ""
-"Dependiendo del objetivo del módulo de indicadores, incite a los "
-"participantes a participar."
+msgstr "Dependiendo del objetivo del módulo de indicadores, incite a los participantes a participar."
 
 #. [administration.helpers.landingPage.chatbot]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.chatbot"
-msgstr ""
+msgstr " "
 
 #. [administration.helpers.landingPage.contact]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.contact"
-msgstr ""
+msgstr " "
 
 #. [administration.helpers.landingPage.data]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.data"
-msgstr ""
+msgstr " "
 
 #. [administration.helpers.landingPage.footer]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.footer"
-msgstr ""
+msgstr " "
 
 #. [administration.helpers.landingPage.header]
 #. defaultMessage is:
 #. Top page header. It contains the consultation's title and subtitle and a button to access to the consultation.
 #: messages.json
 msgid "administration.helpers.landingPage.header"
-msgstr ""
-"Encabezado de la página principal. Contiene el título y el subtítulo de la "
-"consulta y un botón de acceso a la consulta."
+msgstr "Encabezado de la página principal. Contiene el título y el subtítulo de la consulta y un botón de acceso a la consulta."
 
 #. [administration.helpers.landingPage.introduction]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.introduction"
-msgstr ""
+msgstr " "
 
 #. [administration.helpers.landingPage.news]
 #. defaultMessage is:
 #.  
 #: messages.json
 msgid "administration.helpers.landingPage.news"
-msgstr ""
+msgstr " "
```

Comment:
Okay, so these sections weren't translated? Might have to compile the list of empty string sections and resend them for translation.